src/reactable.jsx

```diff
@@ -519,30 +519,30 @@
                     React.Children.forEach(child.props.children, function(descendant) {
                         // TODO
                         /* if (descendant.type.ConvenienceConstructor === Td) { */
-                        if (true) {
```

Comment:
this is an `if (true)` that doesn't have an else case, is it actually being used for anything??